docs/rest.md

```diff
@@ -247,6 +247,14 @@ Which channel to search in.
    derived by Geo IP lookup.
 
 ## /v2/snaps
+
+### Parameters:
+
+#### `select`
+
+Filter from the given selection. Currently only limiting to refreshable
+snaps is supported via the `refresh` key.
```

Comment:
Idle thought (please don't kill me): throughout the review I've been wondering if this is best suited for list (/v2/snaps) or for find (/v2/find). The semantics would be exactly the same and all: same output (meaning, `find --refresh` would return what you have in this branch already), similar Client interface (`cli.Find(&FindOptions{Refresh: true})`), same API (`select=refresh`), etc. The only reason for the change is that `/v2/find` has been about looking for things in the store that we don't have at hand, while `/v2/snaps` became the way to talk about what we have locally. If we draw the line that way, it's awkward to have an option that changes that behavior and suddenly makes /v2/snaps pause.